smithy-model/src/test/resources/software/amazon/smithy/model/loader/valid/elided-members.smithy

```diff
@@ -101,6 +100,7 @@ operation ThrowAway {
         id: String
         property: String
     }
-    output := {}
+    output := {
```

Comment:
This is going to be _very_ common, and I think readers would prefer it stay one line. Is that possible?